src/client/pages/Inkomen/Inkomen.tsx

```diff
@@ -77,15 +77,20 @@ export default function Inkomen() {
       ...(bbzItems || []),
     ]
       .map((item) => {
-        const isBbzHistoric = item.about === 'Bbz' && item.steps.some(step => step.id === "besluit");
+        const isBbzHistoric =
+          item.about === 'Bbz' &&
+          item.steps.some((step) => step.id === 'besluit');
         const activeStatusStep =
           // Bbz steps are sorted in reverse because of a Business decision, unknown rationale
           isBbzHistoric ? item.steps[0] : item.steps[item.steps.length - 1];
```

Comment:
```suggestion
        const activeStatusStep = item.steps[item.steps.length - 1];
```
Ik heb er voor gekozen om de stappen te reversen in het statusline component zodat we over alle aanvragen op dezelfde manier kunnen redeneren.